### ISSUE TITLE
Bug: `sort-classes`: Ternary conditions are not detected as dependencies

### ISSUE THREAD

**Issue by hugop95** [azat-io/eslint-plugin-perfectionist#248]
### Describe the bug

See code example

### Code example

All of the examples below do not detect `b` as a dependency of `a` and consider the codes valid.

```ts
class Class {
  a = this.b ? 1 : 0;
  b = true;
}
```
```ts
class Class {
  a = someCondition ? this.b : 0;
  b = true;
}
```

```ts
class Class {
  a = someCondition ? 1 : this.b;
  b = true;
}
```

### ESLint version

8.57.0

### ESLint Plugin Perfectionist version

v3.3.0

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Fixed in e7c113d.
Released in [v3.4.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.4.0).
